validator/docs/schemas.md

```diff
@@ -0,0 +1,163 @@
+    max-replicas: 10  # Minimal number of pos, INTEGER - Required
+    targetCPUUtilizationPercentage: # Percentage of CPU which will trigger a horizontal scaling when it is exceeded, INTEGER, 0 <= x <= 100 - Required
+  another-process: ...
+```
+
+### Crons
+Crons are tasks that run at a specific time or interval. You can use this type of features to automate tasks, commonly as: reporting, backups, maintenance, emails and so on.
+
+You can define the crons information with the following structure.
+
+```yaml
+crons:
+  my-cronjob: # Cronjob to define, it must be the same name as the cronjob defined in processes
+    concurrency_policy: 'Forbid' # Allow, Forbid or Replace concurrency of cronjobs, STRING - Required. [Kubernetes documentation](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy)
+    schedule: '* * * * *' # STRING - Required
+    suspend:              # BOOLEAN
+  another-resource: ...
+```
+
+### Resources
+This structure defines the resources to be allocated for your application.
+
+```yaml
+resources:
+  my-resource:  # Name of the resource to define
+    limit:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    limits:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    request:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    requests:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+  another-resource: ...
+```
+
+
+### TemplateVars
+Define variables used in configuration templates when building the kubernetes configuration before deployment. Here are the available variables:
+
+_Note_: All the variables should be prefixed by `tpl`.
+
+```yaml
+templateVars:         # Variables used in conf templates
+  tplCriticity:       # Define if the app is on the critical path, STRING - high, low, none
+  tplPublicServices:  # Define if the application web is public accessible or not, BOOLEAN
+  tplExpandedTimeout: # Define the timeout on the ingress controller, STRING
+  tplSessionAffinity: # Activate or deactivate session affinity on ingress controller, BOOLEAN
+  tplTerminationGracePeriod: # Define POD termination grace period, INTEGER
+  tplCanary: # Define name for canary, STRING
+```
+
+### Dependencies
+The list of dependencies used by your project.
+
+```yaml
+dependencies:
+  - first-dependency
+  - second-dependency
+  ...
+```
+
+### Public
+Determines if the project is publicly available or not 
+
+```yaml
+public: true 
+```
+
+### Public Aliases
+Define aliases for public access.
+
+```yaml
+public_aliases:
+  - www # STRING
+  - api # STRING
+  ...
+...
+```
+
+### Probes
+
+If you need more documentation on what probes are, visit the [official documentation](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
+
+In this section, you can define some probes Kubernetes for specific containers. Here is the only available probe for the moment:
```

Comment:
```suggestion
You can define some Kubernetes' probes for each process.
```
I we cannot define probes for other processes than web, maybe change it to something like   `You can define probes for the web process. It is currently not possible to define probes for other processes.`